client/src/views/TrackViewer/Viewer.vue

```diff
@@ -176,6 +179,20 @@ export default defineComponent({
       }
     }
 
+    //Selection of next track while seeking to position
+    function handleSelectNext(delta: number) {
+      const newTrack = selectNextTrack(delta);
+      if (newTrack !== null) {
+        selectTrack(newTrack, false);
+        if (selectedTrackId.value !== null) {
+          const track = trackMap.get(selectedTrackId.value);
+          if (track !== undefined) {
+            playbackComponent.value.seek(track.begin);
+          }
```

Comment:
Should track every be undefined under normal operation conditions?  If not, this conditional should be inverted and throw an exception if undefined is found.